docs/content/integrations/snowflake/using-snowflake-with-dagster-io-managers.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: "Integrating Snowflake with Dagster using I/O managers | Dagster Docs"
+description: "Learn to integrate Snowflake with Dagster using a Snowflake I/O manager."
+---
+
+# Integrating Snowflake with Dagster using I/O managers
+
+This tutorial focuses on how to store and load Dagster's [Software-defined Assets (SDAs)](/concepts/assets/software-defined-assets) in Snowflake by using a Snowflake I/O manager. An [**I/O manager**](/concepts/io-management/io-managers) transfers the responsibility of storing and loading DataFrames as Snowflake tables to Dagster.
+
+By the end of the tutorial, you will:
+
+- Configure a Snowflake I/O manager
+- Create a table in Snowflake using a Dagster asset
+- Make a Snowflake table available in Dagster
+- Load Snowflake tables in downstream assets
+
+This guide focuses on storing and loading Pandas DataFrames in Snowflake, but Dagster also supports using PySpark DataFrames with Snowflake. The concepts from this guide apply to working with PySpark DataFrames, and you can learn more about setting up and using the Snowflake I/O manager with PySpark DataFrames in the [Snowflake reference](/integrations/snowflake/reference).
+
+**Prefer to use resources instead?** Unlike an I/O manager, resources allow you to run SQL queries directly against tables within an asset's compute function. Refer to the [Snowlake resource guide](/integrations/snowflake/using-snowflake-with-dagster) for more info.
```

Comment:
Nothing in here has changed - this callout and other links to the resources version of the tutorial are the only updates.